data/water/wetlands/index.html

```diff
@@ -46,31 +34,14 @@ <h3 id="Wetlands">
   </div>
   <div class="grid__col grid__col--12-of-12 package-content">
     <img class="inline-text-left outline" src="{{ "/images/wetlands2.png" | prepend: site.baseurl }}" alt="Wetlands" />
-    <p>Water.Wetlands represents wetland areas in Utah compiled from NWI, USFS, and UGS sources. In addition to the Cowardin fields, data has been attributed by the Utah Geological Survey to provide functional and descriptive codes associated with each wetland class. This data set is actively being maintained by the Utah Geologic Survey.</p>
+    <p>Water.Wetlands is derived from the U.S. Fish and Wildlife Service’s [National Wetlands Inventory](https://www.fws.gov/wetlands/) (NWI). NWI attributes wetlands with codes from the Cowardin Classification System; codes can be interpreted [online](https://www.fws.gov/wetlands/data/wetland-codes.html). Cowardin classes in this derived layer have been split into separate System, Class, and Water Regime fields. Wetlands spatial data were produced from a combination of aerial imagery examination and on-the-ground assessment and are not meant to be used as the basis for a jurisdictional wetland delineation.</p> 
+    <p>The associated metadata are a description of wetland mapping projects, including the scale at which mapping was conducted and the imagery year used for the mapping. Some mapping projects include supplemental map information that describes the projects in more detail. Most mapping in the state of Utah was conducted using imagery from the 1980s; the Utah Geological Survey is working to update wetlands data across the state. The location of recent and ongoing mapping projects can be found in the [Projects Mapper](https://www.fws.gov/wetlands/data/Mapper.html).</p>
     <h5>Related Resources</h5>
     <ul class="dotless">
       <li><a href="https://geology.utah.gov">Utah Geologic Survey</a></li>
     </ul>
-    <p>For questions, contact Ryhan Sempler at the Utah Geological Survey. 801-537-3338.</p>
```

Comment:
I have [Ryhan Sempler](https://github.com/agrc/gis.utah.gov/blob/master/_data/contacts.yml#L302-L306) in the contacts.yml file. I guess I missed this usage. Will you update the contacts.yml file with the correct information.